scripts/reboot

```diff
@@ -9,7 +9,7 @@ REBOOT_CAUSE_FILE="/var/cache/sonic/reboot-cause.txt"
 
 function stop_sonic_services()
 {
-    echo "Stopping syncd..."
+    echo "Stopping syncd process..."
```

Comment:
```suggestion
    echo "Stopping syncd processing..."
```

Reply:
@qiluo-msft: `Stopping syncd process...` is more appropriate.